sdcm/tester.py

```diff
@@ -762,6 +757,33 @@ def clean_resources(self):
         if self.monitors is not None:
             self.monitors.get_backtraces()
             self.monitors.download_monitor_data()
+
+            # upload prometheus data
+            prometheus_folder = glob.glob(os.path.join(self.logdir, '*monitor*'))[0]
+            file_path = os.path.join(self.logdir, os.path.basename(os.path.normpath(self.job.logdir)))
```

Comment:
You create the zip file insider self.logdir,  use the name of last directory name of self.logdir.
but it compress the content of one sub-directory of self.logdir.

How about use the name of the sub-directory as compress file name?
file_path = os.path.join(self.logdir, os.path.basename(os.path.normpath(prometheus_folder)))